proto/common.proto

```diff
@@ -11,6 +11,35 @@ message LatLng {
   }
 }
 
+message LandmarkManeuver {
```

Comment:
ok so we are going to use this on the trip edge and on the directions maneuver. so i remember we needed something to avoid the naming conflict... what other options do we have...

maybe one of these, but other options exist: PathLandmark, RouteLandmark